src/yoti_common/anchor.processor.js

```diff
@@ -55,7 +55,7 @@ module.exports.AnchorProcessor = class AnchorProcessor {
   static processSingleAnchor(anchorObj) {
     let anchorsList = this.getResultFormat();
 
-    if (!anchorObj instanceof Object) {
+    if (!(anchorObj instanceof Object)) {
```

Comment:
Was this a bug?

Reply:
Not a bug, but this was recommended by ESLint.

Reply:
Cool.  It is clearer this way